Lib/python/pyruntime.swg

```diff
@@ -4,6 +4,13 @@
 # include <math.h>
 #endif
 
+#if defined(_MSC_VER) && _MSC_VER >= 1930
```

Comment:
bug #2174 can be observed already with vs 2019 16.11 (not with 16.11.5, but it is present with 16.11.17 - can't say with which update it appeared in between). As consequence, if we want to activate fix for vs 2019 16.11, test on _MSC_VER should rather be >= 1929

Reply:
@montant Please open a PR with your proposed change so it gets tested automatically by CI.